schemas/context/webinteraction.example.1.json

```diff
@@ -1,5 +1,8 @@
 {
   "xdm:type": "other",
   "xdm:URL": "https://www.example.com/products/store/?view=1",
-  "xdm:name": "product store"
+  "xdm:name": "product store",
+  "https://ns.adobe.com/xdm/metrics/web/link-clicks": {
```

Comment:
@cdegroot-adobe, we'll need to decide what we want the namespace for these to be.  I'm not sure having a metrics namespace is even necessary: would we want metric and non-metric fields that have the same name in the same object?  If we choose to have a 'web' namespace, it will be somewhat redundant to the containing object.  So I actually feel like we should just put this directly into the XDM namespace.  @trieloff, what are your thoughts on that?

Reply:
@kstreeter are you saying 
We change:
`https://ns.adobe.com/xdm/metrics/web/link-clicks`
to be:
`https://ns.adobe.com/xdm/link-clicks`
Which then is the same as
`xdm:link-clicks`

Then we need to refactor the metric definitions.

Reply:
I can live with the shorter namespace URI, but it would be a breaking change. So far, this change looks non-breaking.